### DISCUSSION TITLE
Filtering issues with no:project is faulty

### DISCUSSION THREAD

**Discussion by tobiasamft** [community/community#6332] [Projects and Issues]
# The ability to filter issues with `no:project` is faulty.
Some issues without any project assigned are not listed when they had been assigned to a project before.

## References:
The bug has been mentioned earlier here: https://github.com/isaacs/github/issues/1291#issuecomment-9074331

## Steps to reproduce:
- Have at least one open project
- Have some open issues which are not assigned to a project
- Go to an issue and assign a project
- Filter issues with `no:project` -> **correct result**
- Go to the issue and remove the project
- Filter issues with `no:project` -> **incorrect result**
- Go to issue again, assign yourself, clear assignee again
- Filter issues with `no:project` -> **correct result**

### Step 1: have an open project
![1](https://user-images.githubusercontent.com/35033175/137271905-8d8143b0-aba5-4762-bda4-4372723360b1.png)

### Step 2: filter issues with `no:project`
![2](https://user-images.githubusercontent.com/35033175/137271913-99cbaf75-eb57-494c-bc70-8254350ea365.png)

### Step 3: go to issue and assign project
![3](https://user-images.githubusercontent.com/35033175/137271925-a07c9700-8489-429b-bc8d-19f5a0c27e92.png)

### Step 4: filter issues with `no:project`
![4](https://user-images.githubusercontent.com/35033175/137271935-82161652-eecf-49d5-be64-bf5f18d76ea1.png)

### Step 5: go to issue and remove project
![5](https://user-images.githubusercontent.com/35033175/137271940-f3870fad-9e18-4b36-aea0-7dc3a081c23a.png)

### Step 6: filter issues with `no:project` -> **faulty**: issue is missing
![6](https://user-images.githubusercontent.com/35033175/137271949-f03711d8-ad10-47e9-8d22-e86a952dc94a.png)

### Step 7: go to issue again, assign yourself, clear assignee again
![7](https://user-images.githubusercontent.com/35033175/137271959-ebaf6d13-23c8-46eb-87e9-b552ecd704c4.png)

### Step 8: filter issues with `no:project` -> issue can be seen again
![8](https://user-images.githubusercontent.com/35033175/137271972-2da5ceef-17c9-419e-b503-21b0878c25aa.png)

**Comment by hellwolf** [comment]
yes, please someone help

**Comment by hellwolf** [comment]
yes, please someone help

**Comment by canalnoises** [comment]
It would be really good to get this fixed. We have a repository with ~600 issues and it seems that 29 of them aren't currently assigned to a project. With `no:project` not working properly we have essentially no way to identify those 29 missing issues.

**Comment by randori-ddejohn** [comment]
Cannot currently set up an auto-add project workflow because `no:project` is an invalid field name:
![image](https://github.com/community/community/assets/108751396/1a0dfc6e-2a53-4117-8ec0-1e899b93f954)

**Comment by brandong954** [comment]
Still not able to filter ALL issues with `no:project`. Is this being addressed, or is there a better way to find all issues with no associated project?
